tests/roxctl/bats-tests/local/deployment-bundles-psps.bats

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bats
+
+load "../helpers.bash"
+
+out_dir=""
+
+setup_file() {
+    # remove binaries from the previous runs
+    [[ -n "$NO_BATS_ROXCTL_REBUILD" ]] || rm -f "${tmp_roxctl}"/roxctl*
+    echo "Testing roxctl version: '$(roxctl-release version)'" >&3
+
+    command -v grep || skip "Command 'grep' required."
```

Comment:
I can't imagine a system without `grep`. I'd suggest to remove this line. Up to you.
```suggestion
```

Reply:
Happy to remove it. Added this for consistency reasons (other files contain it).